client/src/components/doctrack/DoctrackSus.svelte

```diff
@@ -0,0 +1,11 @@
+<script>
+    const logoAngry = new URL('../../assets/logo/doctrack-sus.svg', import.meta.url);
+</script>
+
+<img src="{logoAngry.pathname}" alt="Doctrack sus logo" />

```

Comment:
This should be its own component in `components/icons/` instead (without a `<style>` tag for maximal flexibility).

Reply:
Technically not an icon; it's an image